qmarkdowntextedit.cpp

```diff
@@ -160,6 +160,14 @@ void QMarkdownTextEdit::setHighlightingEnabled(bool enabled) {
     }
 }
 
+/**
+ * @brief Returns whetever highlighting is enabled
+ * @return Returns true if highlighting is enabled, oterwise false
```

Comment:
I guess you meant `otherwise`, did you?

Reply:
Oh, yes I did 😅 I'm from Germany and I'm still learning English...

It's time for me to install the Spellchecker plugin, I guess 😅

Reply:
CLion has that built-in. 😉 
Qt Creator is more "meh" in that regard...

In the browser, I like https://languagetool.org/ as an extension.